annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastPackageResolver.java

```diff
@@ -53,4 +54,53 @@ private static String getParentPackage(String packageString) {
         }
         return packageString.substring(0, lastDotIndex);
     }
+
```

Comment:
I don't think the code you've added should be in this class. I'd create new class for this logic. FastFieldsResolver with method resolveFields(FastTypeElement)